doc/STATIC_SAFETY.md

```diff
@@ -416,6 +416,105 @@ code where a field definition in a subclass shadows the field
   allocated.  Users should prefer explicit getters and setters in such
   cases.  See [issue 52](https://github.com/dart-lang/dev_compiler/issues/52).
 
+## Optional Features
+
+### Disable implicit casts
+
+This is an optional feature of strong mode. It disables implicit down casts. For example:
+
+```dart
+main() {
+  num n = 0.5;
+  int x = n; // error: invalid assignment
+  int y = n as int; // ok at compile time, might fail when run
+}
+```
+
+Casts from `dynamic` must be explicit as well:
+
+```dart
+main() {
+  dynamic d = 'hi';
+  int x = d; // error: invalid assignment
+  int y = d as int; // ok at compile time, might fail when run
+}
+```
+
+Configure this for your project by editing .analysis_options:
+
+```yaml
+analyzer:
+  strong-mode:
+    implicit-casts: False
+```
+
+Or pass `--no-implicit-casts` to Dart Analyzer:
+
+```
+dartanalyzer --strong --no-implicit-casts my_app.dart
+```
+
+### Disable implicit dynamic
+
+This is an optional feature of analyzer, intended primarily for use with strong mode's inference.
+It rejects implicit use of `dynamic`, requiring it to be written explicitly. For example:
```

Comment:
You say this below, but worth being really clear up front that this only rejects things that are still dynamic after inference.  e.g.  "It rejects implicit uses of `dynamic` that strong mode inference fails to fill in with a concrete type, ensuring that all types are either successfully inferred or explicitly written".